controllers/submariner/submariner_controller_test.go

```diff
@@ -20,78 +20,23 @@ package submariner_test
 
 import (
 	"context"
-	"fmt"
 	"reflect"
-	"strconv"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/submariner-io/admiral/pkg/log/kzerolog"
-	"github.com/submariner-io/admiral/pkg/syncer/broker"
-	submariner_v1 "github.com/submariner-io/submariner-operator/api/submariner/v1alpha1"
-	submarinerController "github.com/submariner-io/submariner-operator/controllers/submariner"
-	"github.com/submariner-io/submariner-operator/pkg/discovery/network"
+	operatorv1 "github.com/submariner-io/submariner-operator/api/submariner/v1alpha1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/client-go/kubernetes/scheme"
-	controllerClient "sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/client/fake"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
 const (
-	submarinerName          = "submariner-gateway"
+	submarinerName          = "submariner"
 	submarinerNamespace     = "submariner-operator"
```

Comment:
These 2 constants are defined in `internal/constants/constants.go` as well. That could be used here.

Reply:
These predated this PR. But the controller code does not/should not rely on the global constants (the namespace at least) so neither should the test code. In fact we should use values that differ from the global constants to ensure the controller code does not use them. I'll make that change.